experimental/routes_refactor/demo/scripts/port-forward-bookbuyer-envoy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Why are we adding port forwarding scripts again done these already exist in `./scripts`

Reply:
These port-forwarding scripts are slightly different as they use an additional label selector (version) for the bookstore pods. It seemed convenient to put all the relevant scripts in one place (in the experimental directory) to easy access them while running the routes refactor demo in the experimental/routes_refactor directory. I was thinking once this feature is the default, we could get rid of this directory altogether. Let me know if you prefer we go a different route.

Reply:
I didn't notice the version, I think its okay to keep this directory for now.